errbot/backends/slack.py

```diff
@@ -565,6 +565,24 @@ def send_message(self, mess):
                 "to %s: %s" % (to_humanreadable, mess.body)
             )
 
+    def send_stream_request(self, identifier, fsource, name=None, size=None, stream_type=None):
+        """Starts a file transfer. For Slack, the size and stream_type are unsupported"""
+        name = name if name else 'file'
+        stream = Stream(identifier, fsource, name, size, stream_type)
+        log.debug('Initiating upload of {0}'.format(name))
+        resp = self.api_call('files.upload', data={
```

Comment:
I'm currently testing this, and it looks like the WorkRequest never gets serviced. There's no error in the log, just an enqueued WorkRequest and then silence. I'm really not familiar with this part of errbot - is there any documentation or guidance in how this subsystem works?